### PR TITLE
Update production-offline-deployment-using-tiup.md link

### PR DESCRIPTION
**Pull request by uglyengineer**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: production-offline-deployment-using-tiup.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ title: 使用 TiUP 离线部署 TiDB 集群
 
 ### 方式一：下载官方 TiUP 离线组件包
 
-在 [官方下载页面](https://pingcap.com/download) 选择对应版本的离线镜像包。
+在 [官方下载页面](https://pingcap.com/download-cn/) 选择对应版本的离线镜像包。
```
Comment (yikeke) [inline production-offline-deployment-using-tiup.md:13]: @lonng @lucklove 这句话的描述需不需要更新下？说明下选择哪个？⬇️

image

Comment (uglyengineer) [reply production-offline-deployment-using-tiup.md]: 我加了描述，选择 server 包下载

File: production-offline-deployment-using-tiup.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ title: 使用 TiUP 离线部署 TiDB 集群
 
 ### 方式一：下载官方 TiUP 离线组件包
 
-在 [官方下载页面](https://pingcap.com/download) 选择对应版本的离线镜像包。
+在 [官方下载页面](https://pingcap.com/download-cn/) 选择对应版本的 TiDB server 离线镜像包。
```
Comment (yikeke) [inline production-offline-deployment-using-tiup.md:13]:
```suggestion
在 [官方下载页面](https://pingcap.com/download-cn/) 选择对应版本的 TiDB server 离线镜像包（包含 TiUP 离线组件包）。
```
@lichunzhu PTAL

Comment (lichunzhu) [reply production-offline-deployment-using-tiup.md:13]: LGTM

Comment (uglyengineer) [reply production-offline-deployment-using-tiup.md]: @lonng PTAL

**Pull request merged**